Microsoft.AspNetCore.Components.Web/Microsoft.AspNetCore.Components.Web/ErrorEventArgs.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.AspNetCore.Components.Web
+{
+	/// <summary>
+	/// Supplies information about an error event that is being raised.
+	/// </summary>
+	public class ErrorEventArgs : EventArgs
+	{
+		/// <summary>
+		/// Gets a a human-readable error message describing the problem.
+		/// </summary>
+		public string? Message
```

Comment:
Hmmm. According to the spec, these values shouldn't be nullable ([ref](https://html.spec.whatwg.org/multipage/webappapis.html#the-errorevent-interface)). Not sure if we've made a decision to be different here.